DC-SLES-container-guide

```diff
@@ -0,0 +1,16 @@
+## ---------------------------- 
+## Doc Config File for SLES
+## Container Guide
+## ----------------------------
+##
+## Basics
+MAIN="book_sles_container.xml"
+#ROOTID=book-sle-container
```

Comment:
I think the goal was to integrate this into `MAIN.SLEDS.xml` as early as possible.
Please XInclude it there & use it as MAIN, then correctly apply the ROOTID